### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by mckent05**
In this Pull request, I have added the following:

- A html file containing hello world
- A css file used i styling the font of the hello world
- A readme.md file that describes the project and how you can clone this project.
- A .gitignore file that ignores node_modules and test.md files

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (arslanbisharat) [inline .github/workflows/linters.yml:48]: - [ ] Please make sure you don't have any failing linter rule. 


You can find this as [General Requirement ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#general-requirements).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+HELLO WORLD PROJECT
+This is a simple hello world project for Module 1 Weekday 2
+
+For this project I used:
+HTML: to add an h1 tag containing hello world
+CSS: to style the background color of my h1 tag to my favorite color and also chnage the color of the text.
```
Comment (arslanbisharat) [inline README.md]: - [ ] Please add a descriptive Readme to your project. Please use [this ](https://github.com/microverseinc/readme-template) template but remember to customize it to your project.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
- [ ] The Pull Request should have a descriptive title and message. Please read [this ](https://zellwk.com/blog/edit-pull-request/) article to help you with this concern. 



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .github/workflows/linters.yml (hunk 1 above)
Comment (mckent05) [reply .github/workflows/linters.yml:48]: I do not understand why stylelint keeps giving me indentation errors, I have checked online and the issue still persists please help

**Comment by mckent05** [comment]
I have made changes to all your corrections and my project is ready for review.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+> This is a simple Hello Miroverse Project.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML- Hypertext Markup Language
+- CSS - Cascading Style Sheets
+
```
Comment (BrianSammit) [inline README.md:17]: - [ ] It's so much better and professional if you provide steps about how to clone and start your project. Please provide an explanation about how to do this.
Remember that not only developers are going to see your repo so it so much better if you explain to non developer people how to run your project. 
 👍 🥇

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED :red_circle: 🛑 

Hi there, 👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] As the previous reviewer ask for, It would be better if you provide a pull request description. This will help your team (In this case the reviewer) know what are you asking to merge into the main branch. You could write something like 
```
In this pull request, we added:
- A hello.html file that contains Hello world 
- A hello.css file that contains .....
- A .gitignore that contains .....

``` 

- [ ] Check my inline comment below 👇 

----------
Please fix these minor errors and sen it again for another review :+1: :100: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 



**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by afanimartin — APPROVED** [review]
Hi @mckent05, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**